downstream/modules/platform/ref-access-control-requirements.adoc

```diff
@@ -0,0 +1,170 @@
+
+[id="ref-access-control-requirements_{context}"]
+
+= Access control requirements
+
+[role="_abstract"]
+
+{PlatformName} uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by a firewall.
+
+The following tables provide the default {PlatformName} destination ports required for each application.
+
+.PostgreSQL
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Remote access during installation
+|5432
+|TCP
+|Postgres
+a|Default port
+
+ALLOW connections from controller(s) to database port
```

Comment:
ENABLE not allow, throughout.

Reply:
@ianf77 Is this a preference or is there a style guideline? I'm curious as I would use "allow" when talking about ports. When you open a port you allow traffic through it, whereas enable implies you do something that turns on the ability to send traffic through a port. I feel there is a semantic change in meaning that doesn't really work.

Reply:
Then again it could just be me and the fact I haven't had coffee yet. :wink:

Reply:
IBM Style guide says : Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a
direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

Reply:
Editors at a previous company would ruthlessly purge all appearance of allow, so I learned to avoid it.

Reply:
Thanks for clarifying, Ian. My 2c is that doesn't necessarily apply in this case. In general that's good advice but it seems like it's more in the spirit of "focusing on the task, not the tools".

It's still my view that `s/allow/enable` misses some nuances. When you open a port on the network, you allow traffic through that port. In this scenario the user does not enable any functionality.

I also think the convention with inbound/outbound ports is to "allow traffic" so changing to "enable" just doesn't sound correct to a technical audience.